open_spiel/python/pybind11/python_policy.h

```diff
@@ -0,0 +1,49 @@
+
+#ifndef OPEN_SPIEL_PYTHON_POLICY_H
+#define OPEN_SPIEL_PYTHON_POLICY_H
+
+#ifndef SINGLE_ARG
+   #define SINGLE_ARG(...) __VA_ARGS__
+#endif
```

Comment:
Can we confine this to .cc files only?

Reply:
whoops, this was supposed to be in the .cc anyway